x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/EqlFunctionRegistry.java

```diff
@@ -37,6 +38,7 @@ public EqlFunctionRegistry() {
                 def(EndsWith.class, EndsWith::new, "endswith"),
                 def(IndexOf.class, IndexOf::new, "indexof"),
                 def(Length.class, Length::new, "length"),
+                def(ToNumber.class, ToNumber::new, "number"),
```

Comment:
I think we should add a new `FunctionDefinition[]` array for math/numerical functions.

Reply:
agreed. @aleksmaus has one in his PR. I wasn't sure if this was math or still string, since it's parsing from strings